docs/generics.rst

```diff
@@ -250,3 +250,40 @@ want to override; like so::
 You can also use the Django ``method_decorator`` utility to wrap methods that
 have ``message`` as their first positional argument - note that it won't work
```

Comment:
This is not part of this PR. I can change it but only if @andrewgodwin will agree.